addons/oceanbase-ce/templates/_helpers.tpl

```diff
@@ -73,13 +62,160 @@ Generate scripts configmap
 {{/*
 Generate reloader scripts configmap
 */}}
-{{- define "oceanbase.extend.reload.scripts" -}}
+{{- define "oceanbase-ce.extend.reload.scripts" -}}
 {{- range $path, $_ :=  $.Files.Glob "reloader/**" }}
 {{ $path | base }}: |-
 {{- $.Files.Get $path | nindent 2 }}
 {{- end }}
 {{- end }}
 
-{{- define "oceanbase-release.name" }}
-{{- print "ob-ce" }}
-{{- end }}
\ No newline at end of file
+
+{{- define "oceanbase-ce.compDefName" -}}
+oceanbase-ce-{{ .Chart.Version }}
+{{- end -}}
+
+{{- define "oceanbase-ce.componentDefNamePrefix" -}}
```

Comment:
use regexp pattern `oceanbase-ce-` would be better, follow the API suggestion, check them all

Reply:
fixed